### PR TITLE
[NO JIRA] Passing the correct args, instead of the exceptions

### PR DESCRIPTION
**Pull request by artsmandev**
I have suggestions on `cosmetic changes` and `typo fixes`:

1. The logs are expecting arguments, but, they're passing an exception instead of the correct args.

- [x]  Use the following formatting style: SonarSource/sonar-developer-toolset
- [x] Provide a unit test for any code you changed
- [ ] If there is a JIRA ticket available, please make your commits and pull request starting with the ticket ID (SONAR-XXXX)

Thank you.

### CODE REVIEW

File: server/sonar-ce/src/main/java/org/sonar/ce/taskprocessor/CeProcessingSchedulerImpl.java [hunk 1]
```diff
@@ -111,7 +111,7 @@ public void hardStopScheduling() {
     try {
       waitForInProgressWorkersToFinish(350);
     } catch (InterruptedException e) {
-      LOG.debug("Grace period of hard stop has been interrupted: {}", e);
+      LOG.debug("Grace period of hard stop has been interrupted: {}", e.getMessage());
```
Comment (pierre-guillot-gh) [inline server/sonar-ce/src/main/java/org/sonar/ce/taskprocessor/CeProcessingSchedulerImpl.java:114]:
```suggestion
      LOG.debug("Grace period of hard stop has been interrupted: {}", e.getMessage(), e);
```

File: server/sonar-db-dao/src/main/java/org/sonar/db/DBSessionsImpl.java [hunk 2]
```diff
@@ -69,7 +69,7 @@ private static void ensureAutoCommitFalse(DbSession dbSession) {
         checkState(!((boolean) f.get(sqlSession)), "Autocommit must be false");
       }
     } catch (NoSuchFieldException | IllegalAccessException e) {
-      LOG.debug("Failed to check the autocommit status of SqlSession: {}", e);
+      LOG.debug("Failed to check the autocommit status of SqlSession: {}", e.getMessage());
```
Comment (pierre-guillot-gh) [inline server/sonar-db-dao/src/main/java/org/sonar/db/DBSessionsImpl.java:72]:
```suggestion
      LOG.debug("Failed to check the autocommit status of SqlSession: {}", e.getMessage(), e);
```

File: server/sonar-webserver-core/src/main/java/org/sonar/server/qualitygate/ProjectsInWarningDaemon.java [hunk 3]
```diff
@@ -104,7 +104,7 @@ private Runnable countProjectsInWarning() {
       try (DbSession dbSession = dbClient.openSession(false)) {
         updateProjectsInWarningInDb(dbSession, nbProjectsInWarning);
       } catch (Exception e) {
-        LOG.error("Error updating number of projects in warning: {}", e);
+        LOG.error("Error updating number of projects in warning: {}", e.getMessage());
```
Comment (pierre-guillot-gh) [inline server/sonar-webserver-core/src/main/java/org/sonar/server/qualitygate/ProjectsInWarningDaemon.java:107]:
```suggestion
        LOG.error("Error updating number of projects in warning: {}", e.getMessage(), e);
```

File: server/sonar-webserver-core/src/main/java/org/sonar/server/telemetry/TelemetryDaemon.java [hunk 4]
```diff
@@ -114,7 +114,7 @@ private Runnable telemetryCommand() {
           updateTelemetryProps(now);
         }
       } catch (Exception e) {
-        LOG.debug("Error while checking SonarQube statistics: {}", e);
+        LOG.debug("Error while checking SonarQube statistics: {}", e.getMessage());
```
Comment (pierre-guillot-gh) [inline server/sonar-webserver-core/src/main/java/org/sonar/server/telemetry/TelemetryDaemon.java:117]:
```suggestion
        LOG.debug("Error while checking SonarQube statistics: {}", e.getMessage(), e);
```

File: server/sonar-webserver-webapi/src/main/java/org/sonar/server/health/DbConnectionNodeCheck.java [hunk 5]
```diff
@@ -50,7 +50,7 @@ private boolean isConnectedToDB() {
     try (DbSession dbSession = dbClient.openSession(false)) {
       return dbSession.getMapper(IsAliveMapper.class).isAlive() == IsAliveMapper.IS_ALIVE_RETURNED_VALUE;
     } catch (RuntimeException e) {
-      LOGGER.trace("DB connection is down: {}", e);
+      LOGGER.trace("DB connection is down: {}", e.getMessage());
```
Comment (pierre-guillot-gh) [inline server/sonar-webserver-webapi/src/main/java/org/sonar/server/health/DbConnectionNodeCheck.java:53]:
```suggestion
      LOGGER.trace("DB connection is down: {}", e.getMessage(), e);
```

**Review by pierre-guillot-gh — CHANGES_REQUESTED** [review]
Hey, thanks for your contribution, I think that makes sense. I do have one change request: could you please adjust and log both the message and the exception stacktrace? That way, we will have all the information we need (specifically important for debug/trace logs).

File: server/sonar-ce/src/main/java/org/sonar/ce/taskprocessor/CeProcessingSchedulerImpl.java (hunk 1 above)
Comment (artsmandev) [reply server/sonar-ce/src/main/java/org/sonar/ce/taskprocessor/CeProcessingSchedulerImpl.java]: I did it, 🤝.
Thank you for the feedback @pierre-guillot-gh!

File: server/sonar-db-dao/src/main/java/org/sonar/db/DBSessionsImpl.java (hunk 2 above)
Comment (artsmandev) [reply server/sonar-db-dao/src/main/java/org/sonar/db/DBSessionsImpl.java]: I did it, 🤝.
Thank you for the feedback @pierre-guillot-gh!

File: server/sonar-webserver-core/src/main/java/org/sonar/server/telemetry/TelemetryDaemon.java (hunk 4 above)
Comment (artsmandev) [reply server/sonar-webserver-core/src/main/java/org/sonar/server/telemetry/TelemetryDaemon.java:117]: I did it, 🤝.
Thank you for the feedback @pierre-guillot-gh!

**Comment by pierre-guillot-gh** [comment]
Thanks for your contribution! As this git repository is a clone of a private one, I will cherry-pick your commit to the private one and it will be mirrored here in a few days. You will still appear as the author of the commit 👍

**Pull request closed without merging**